main/.vuepress/themeConfig/nav.js

```diff
@@ -66,6 +66,11 @@ module.exports = [
       }
     ],
   },
+  {
+    text: 'Glossary',
+    ariaLabel: 'Glossary Menu',
+    link: '/glossary'
```

Comment:
I think this needs to be:
```suggestion
    link: '/glossary/'
```
So that it goes to the index.html, not a file named glossary.html